### ISSUE TITLE
Add indication of whether files include the @flow annotation

### ISSUE THREAD

**Issue by ryan953** [rpl/flow-coverage-report#12]
When incrementally adding flow coverage to an existing project it is recommended to add the @flow annotation to one file, run `flow` to see a list of errors and proceed to fix those errors before starting on the next file. See https://flowtype.org/docs/existing.html. However it's not reasonable to expect each file will achieve 100% coverage after it has been touched because not all of the dependencies will have coverage either.

Tp support this workflow it would be helpful if flow-coverage-report would indicate not just what percent of a file is covered by flow, but which files have `/* @flow */` or `/* @flow weak */` annotations. This would make it easier to ensure that all statements are covered, but also that errors/warnings are emitted by flow when they should be.

**Comment by rpl** [comment]
@ryan953 That's a good point

and it looks like the `flow ast` command can provide some help to extract this information in a more reliable way, e.g.:

```
$ flow ast test.js
{
  "type" : "Program",
  "body" : [
  ....
  ],
  "comments" : [{
     "range" : [0, 16],
     "loc" : { ... },
     "type" : "Block",
     "value" : " @flow weak "
  } ]
}
```

**Comment by ryan953** [comment]
I started looking at that AST too :).

Flow adds some extra logic to only check the first 9 lines for @flow and other comments/statements. I'm not Sure about all the syntax but checkout https://github.com/facebook/flow/blob/d2930e135f9c34a89a226c2ca36eb4bcab7b59db/src/parsing/docblock.ml for where the limits are. 

I think it would be better for flow itself to report the status along with all the files it sees, maybe as an addition to 'flow ls'. HHVM supports this with one of the 'hh' commands. 
In the mean time though a method to validate existing @flow annotations would be cool, could even be reused as a lint rule and maybe more.

**Comment by migueloller** [comment]
FWIW, it would be useful to also have a flag that will run the coverage report only on files that have teh `@flow` pragma.

**Comment by sheldonkwok** [comment]
It would also be useful if the coverage test could ignore files that don't have the `// @flow` designation because they are not ready to be tested yet.

**Comment by rpl** [comment]
Fixed by #27 (merged as #55)

In the current form it reports the flow preamble annotations but doesn't change the behavior of the coverage return status (and how the coverage totals are computed).
In follow ups issues we can define a different behavior (and related cli options) as being discussed in the comments of this issue and #27.
